armada/handlers/armada.py

```diff
@@ -102,16 +108,17 @@ def pre_flight_ops(self):
                         try:
                             LOG.info('Cloning repo: %s', location)
                             repo_dir = git.git_clone(location, reference)
-                        except Exception as e:
-                            raise ValueError(e)
+                        except Exception:
```

Comment:
can this be changed to catch the specific error thrown by pygit2? I believe it would be `except pygit2.GitError:`